CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 `relude` uses [PVP Versioning][1].
 The changelog is available [on GitHub][2].
 
+## 0.X.0
```

Comment:
We've already planned for the next milestone and we know its version now, so we can put in the CHANGELOG as well 🙂

```suggestion
## Unreleased: 0.8.0.0
```